.github/workflows/test_api.yml

```diff
@@ -0,0 +1,24 @@
+name: Test API using Pytest
+
+on:
+  pull_request:
+
+jobs:
+  test_api:
+    env:
+      SECRET_KEY: ${{ secrets.SECRET_KEY }}
+      DEV_DB_CONN_STRING: ${{secrets.DEV_DB_CONN_STRING}}
```

Comment:
i have added this as an org level secret with the connection string for the new DO data_sources_app_v2 user